File: src/main/java/no/nav/service/pensjon/regler/to/RevurderingOvergangskullRequest.java
package no.nav.service.pensjon.regler.to;

import java.util.Date;
import java.util.Vector;

import no.nav.domain.pensjon.regler.beregning2011.AfpLivsvarig;
import no.nav.domain.pensjon.regler.beregning2011.SisteAldersberegning2011;
import no.nav.domain.pensjon.regler.grunnlag.DelingstallUtvalg;
import no.nav.domain.pensjon.regler.grunnlag.ForholdstallUtvalg;
import no.nav.domain.pensjon.regler.grunnlag.InfoPavirkendeYtelse;
import no.nav.domain.pensjon.regler.kode.ResultatTypeCti;
import no.nav.domain.pensjon.regler.krav.Kravhode;
import no.nav.domain.pensjon.regler.vedtak.VilkarsVedtak;

public class RevurderingOvergangskullRequest extends ServiceRequest {
	

	private Kravhode kravhode;
	private Vector<VilkarsVedtak> vilkarsvedtakListe ;
	private InfoPavirkendeYtelse infoPavirkendeYtelse;
	private DelingstallUtvalg delingstallUtvalg;
	private ForholdstallUtvalg forholdstallUtvalg;
	private Date virkFom;
	private SisteAldersberegning2011 sisteAldersBeregning;
	private double nettoUPellerGJP;
	private ResultatTypeCti nettoUPellerGJPresultatType;
	private AfpLivsvarig afpLivsvarig;

	
}

File: src/main/kotlin/no/nav/pensjon/regler/to/RevurderingOvergangskullRequest.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.beregning2011.AfpLivsvarig
import no.nav.pensjon.regler.domain.beregning2011.SisteAldersberegning2011
import no.nav.pensjon.regler.domain.grunnlag.DelingstallUtvalg
import no.nav.pensjon.regler.domain.grunnlag.ForholdstallUtvalg
import no.nav.pensjon.regler.domain.grunnlag.InfoPavirkendeYtelse
import no.nav.pensjon.regler.domain.kode.ResultatTypeCti
import no.nav.pensjon.regler.domain.krav.Kravhode
import no.nav.pensjon.regler.domain.vedtak.VilkarsVedtak
import java.util.*

class RevurderingOvergangskullRequest : ServiceRequest() {
    var kravhode: Kravhode? = null
    var vilkarsvedtakListe: Vector<VilkarsVedtak> = Vector()
    var infoPavirkendeYtelse: InfoPavirkendeYtelse? = null
    var delingstallUtvalg: DelingstallUtvalg? = null
    var forholdstallUtvalg: ForholdstallUtvalg? = null
    var virkFom: Date? = null
    var sisteAldersBeregning: SisteAldersberegning2011? = null
    var nettoUPellerGJP = 0.0
    var nettoUPellerGJPresultatType: ResultatTypeCti? = null
    var afpLivsvarig: AfpLivsvarig? = null

}